src/Eshopworld.Core/Eshopworld.Core.csproj

```diff
@@ -1,8 +1,9 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
 
-    <Version>1.0.2</Version>
+    <Version>1.0.3</Version>
```

Comment:
True, the reason why I didn't bump major was because this is an internal package, only referenced by other Eshopworld packages. Where we are introducing breaking API is at the Eshopworld.Messaging level. I usually bump major when we do a radical change on the core package structure.